src/families/ethereum/EthereumCustomFees.js

```diff
@@ -0,0 +1,150 @@
+/* @flow */
+import React, { useState, useCallback } from "react";
+import { StyleSheet, View } from "react-native";
+import { BigNumber } from "bignumber.js";
+import invariant from "invariant";
+import { useSelector } from "react-redux";
+import { useTheme } from "@react-navigation/native";
+import { inferDynamicRange } from "@ledgerhq/live-common/lib/range";
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import { i18n } from "../../context/Locale";
+import { accountScreenSelector } from "../../reducers/accounts";
+import EditFeeUnitEthereum from "./EditFeeUnitEthereum";
+import SectionSeparator from "../../components/SectionSeparator";
+import Button from "../../components/Button";
+import EthereumGasLimit from "./SendRowGasLimit";
+
+import type { RouteParams } from "../../screens/SendFunds/04-Summary";
+
+type Props = {
+  navigation: any,
+  route: { params: RouteParams },
+};
+
+const options = {
```

Comment:
same as previous component prefer declaring the options in the navigator directly